app/components/channel_list/channel_list.js

```diff
@@ -16,7 +16,11 @@ import deepEqual from 'deep-equal';
 const Styles = StyleSheet.create({
     container: {
         flex: 1,
-        marginTop: 20
+        ...Platform.select({
```

Comment:
It can be a separate PR, but we might want to move this to the `RootLayout` so that it's applied to every scene automatically